src/stackedBars.ts

```diff
@@ -89,6 +134,25 @@ export function renderStackedBars(pareto: Pareto, settings: Settings) {
         markingSelector: ".in-bar"
     });
 
+
+console.log(barCounter)
+    function selectedBars(){

```

Comment:
There's an issue with the logic when selecting bars in color by. 

![image](https://user-images.githubusercontent.com/63287192/207535783-5c1f1ee3-fc1b-4d6c-b960-c1148e43927b.png)